charts/ocis/values.yaml

```diff
@@ -4,9 +4,9 @@ image:
   # -- Image repository
   repository: owncloud/ocis
   # -- Image tag. Defaults to the chart's appVersion.
-  tag: "latest"
+  tag: ""
```

Comment:
Don't we have a tag we could use?

Reply:
tag defaults to the chart's appVersion. Therfore we don't need a tag

Reply:
Ah right, missed that.